view/global.css

```diff
@@ -399,14 +399,9 @@ a {
   color: #00a700;
 }
 .federation-graph {
-  width: 400px;
-  height: 400px;
-  float: right;
```

Comment:
why was the float removed?

Reply:
Because it doesn't look good right now. 😛

Reply:
You have redesigned the page? The floating was added to avoid to much dead space aside of the graph.

Reply:
My current version has dead space only on the left of the graph, so I centered it horizontally instead.

Reply:
So now there is dead space left and right aside of the graph and aside of the project listing. Well if you like it that way better ;-)

I would prefer the heading of the page being the first thing on the page as well. The way it looks now, for me, is that the graph is totally unrelated to the rest of the page. Before the stats were aside of the graph so they were related.

Reply:
Ah, but it never looked this way on frio, hence the confusion between us.

Reply:
Oh, that is bad. Anyhow the way it looks now is worse then before IMO I'll revert this merge maybe ;-)

Reply:
Please don't, this wasn't a display merge, but a library move. If you want to improve the display in vier, please add your changes to the theme.